ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -80,127 +79,57 @@ import           Test.Ouroboros.Consensus.ChainGenerator.Params (Delta (Delta))
-
-instance CondenseList TipPoint where
-  condenseList = condenseListWithPadding PadRight
-
--- | The latest header that should be sent to the client by the ChainSync server
--- in a tick.
-newtype HeaderPoint =
-  HeaderPoint (WithOrigin (Header TestBlock))
-  deriving (Eq, Show)
-
-instance Condense HeaderPoint where
-  condense (HeaderPoint header) = terseWithOrigin terseHeader header
-
-instance CondenseList HeaderPoint where
-  condenseList = condenseListWithPadding PadRight
-
--- | The latest block that should be sent to the client by the BlockFetch server
--- in a tick.
-newtype BlockPoint =
-  BlockPoint (WithOrigin TestBlock)
-  deriving (Eq, Show)
-
-instance Condense BlockPoint where
-  condense (BlockPoint block) = terseWithOrigin terseBlock block
-
-instance CondenseList BlockPoint where
-  condenseList = condenseListWithPadding PadRight
-
--- | The set of parameters that define the state that a peer should reach when it receives control
--- by the scheduler in a single tick.
---
--- REVIEW: I find this rather poorly named. If it is really what is advertised
--- then isn't it weird to have the fragment in it? If it is the whole internal
--- state of the (online) node, then maybe we can call it that?
-data AdvertisedPoints =
-  AdvertisedPoints {
-    tip    :: TipPoint,
-    header :: HeaderPoint,
-    block  :: BlockPoint
+-- | The state of a peer at a given point in time.
+data NodeState blk =
+  NodeState {
+    nsTip    :: WithOrigin blk,
+    nsHeader :: WithOrigin blk,
+    nsBlock  :: WithOrigin blk
   }
   deriving (Eq, Show)
 
-instance Condense AdvertisedPoints where
-  condense AdvertisedPoints {tip, header, block} =
-    "TP " ++ condense tip ++
-    " | HP " ++ condense header ++
-    " | BP " ++ condense block
+nsTipTip :: AF.HasHeader blk => NodeState blk -> Tip blk
+nsTipTip = withOrigin TipGenesis tipFromHeader . nsTip
+
+instance Condense (NodeState TestBlock) where
+  condense NodeState {nsTip, nsHeader, nsBlock} =
+    "TP " ++ terseWithOrigin terseBlock nsTip ++
+    " | HP " ++ terseWithOrigin terseBlock nsHeader ++
+    " | BP " ++ terseWithOrigin terseBlock nsBlock
 
-instance CondenseList AdvertisedPoints where
+instance CondenseList (NodeState TestBlock) where
   condenseList points =
     zipWith3
       (\tip header block ->
         "TP " ++ tip ++
           " | HP " ++ header ++
           " | BP " ++ block
       )
-      (condenseList $ tip <$> points)
-      (condenseList $ header <$> points)
-      (condenseList $ block <$> points)
-
-genesisAdvertisedPoints :: AdvertisedPoints
-genesisAdvertisedPoints =
-  AdvertisedPoints {
-    tip = TipPoint TipGenesis,
-    header = HeaderPoint Origin,
-    block = BlockPoint Origin
+      (padListWith PadLeft $ map (terseWithOrigin terseBlock . nsTip) points)
+      (padListWith PadLeft $ map (terseWithOrigin terseBlock . nsHeader) points)
+      (padListWith PadLeft $ map (terseWithOrigin terseBlock . nsBlock) points)
```

Comment:
The lists of formerly tip/header/block-points used to be left-aligned (i.e. padded with `PadRight`).
I'm not particularly opinionated on the question, but do you think right-alignment looks better?

Reply:
It was a mistake from my side, I do want those things left-aligned. Done in cbe13492d4c44ede88075990eeb35480f6966de4.